src/ski.go

```diff
@@ -67,9 +68,9 @@ func isValidPlanet(planet Planet) bool {
 	if !ok {
 		switch planet.planetType {
 		case webServer:
-			fmt.Fprintf(os.Stderr, "Usage of ski with web servers is not implemented")
+			planet.outputStruct.output += makeRed("Usage of ski with web servers is not implemented\n")
```

Comment:
How about adding an error field to outputStruct on the planet for error messages. At the end of the execution you will have to mark only one chunk of text red instead of peppering the whole code base with calls to makeRed function.